### PR TITLE
sync package to repo api

### PR DESCRIPTION
**Pull request by huiwq1990**
Signed-off-by: huiwq1990 <huiwq1990@163.com>

/kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage
> /sig storage

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@kadisi PTAL

File: hack/lib/sync-api.sh [hunk 1]
```diff
@@ -26,48 +26,41 @@ echo "git clone"
...
-cp -R yurt-app-manager/pkg/yurtappmanager/apis/* yurt-app-manager-api/pkg/yurtappmanager/apis/
+echo "update apps"
+cp -R yurt-app-manager/pkg/yurtappmanager/apis/apps/* yurt-api/apps/
 # remove controller depends functions
-rm -r yurt-app-manager-api/pkg/yurtappmanager/apis/apps/v1alpha1/defaults.go
+rm -rf yurt-api/apps/v1alpha1/defaults.go
+
+echo "update addtoschema"
+rm -rf yurt-api/addtoscheme_apps_v1alpha1.go
+cp yurt-app-manager/pkg/yurtappmanager/apis/addtoscheme_apps_v1alpha1.go yurt-api/addtoscheme_apps_v1alpha1.go
```
Comment (rambohe-ch) [inline hack/lib/sync-api.sh:46]: maybe `addtoscheme_apps_v1alpha1.go` should be placed under `yurt-api/apps` dir?

Comment (huiwq1990) [reply hack/lib/sync-api.sh:46]: The `addtoscheme_apps_v1alpha1 .go` package should be renamed to apps, it's not alligin with origin project.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**